projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.d.ts

```diff
@@ -933,6 +933,7 @@ interface SohoDataGridStatic {
 
   /** Reference to pager. */
   pager: SohoPagerStatic;
```

Comment:
does this become deprecated then?

Reply:
Sure we call it pagerAPI internally so i copied that over. I wasnt really sure if this was accessed this way or not so i left both. But we can deprecate it ?

Reply:
good question; i was just asking because of two properties that had the same type

Reply:
I'll just mark pager as deprecated in favor of pagerAPI as we have API in a few places

Reply:
Soon...when im done another PR